source/new-ia/by-role/system-admin/reverse-proxy-configuration.rst

```diff
@@ -0,0 +1,75 @@
+:is-up-to-date: True
+:since-version: 4.0.3
+:nosearch:
+
+.. index:: Reverse Proxy Configuration, Configuring Apache Virtual Host, Configuring Apache vhost
+
+.. _newIa-reverse-proxy-configuration:
+
+===========================
+Reverse Proxy Configuration
+===========================
+
+.. version_tag::
+   :label: Since
+   :version: 4.0.3
+
+In this section, we discuss how to configure a reverse proxy for Apache for your
+CrafterCMS authoring and delivery installs.
```

Comment:
... reverse proxy using Apache 2 HTTPd vhost configuration for authoring and delivery.